marklogic/tests/fixtures/base_status.yaml

```diff
@@ -0,0 +1,335 @@
+local-cluster-status:
```

Comment:
The fixtures were useful for understanding the structure of the payloads during development, but I think they might not be needed anymore.